src/ort_genai.h

```diff
@@ -9,6 +9,7 @@
 
 #if __cplusplus >= 202002L
 #include <span>
+#define OGA_USE_SPAN 1
```

Comment:
Should this always be 1?

Reply:
It needs to be conditional in the public header, as users won't have our "span.h" if their compiler doesn't have it.